CONTRIBUTING.md

```diff
@@ -370,7 +370,7 @@ If you have an affiliation to the brand you contributing that allows you to spea
 
 ## Testing Package Locally
 
-* Make sure you have [NodeJS](https://nodejs.org/en/download/) installed. At least version 12.4.0 is required.
+* Make sure you have [NodeJS](https://nodejs.org/en/download/) installed. At least version 12.20.0 is required.
```

Comment:
I'm thinking about this part. The exact version range should be `^12.20.0 || ^14.13.1 || >=16.0.0`.

```suggestion
* Make sure you have [Node.js](https://nodejs.org/en/download/) installed. At least version `^12.20.0 || ^14.13.1 || >=16.0.0` is required.
```